inference-engine/src/mkldnn_plugin/nodes/mkldnn_input_node.h

```diff
@@ -21,19 +21,18 @@ class MKLDNNInputNode : public MKLDNNNode {
     void createPrimitive() override;
     bool created() const override;
 
-    void execute(mkldnn::stream strm) override;
-    void withMeanImage() {
-        isMeanImage = true;
-    }
+    void withMeanImage();
+    const InferenceEngine::Blob::CPtr getConstBlob() const;
+    MKLDNNMemoryPtr getMemoryPtr() const;
 
-    const InferenceEngine::Blob::CPtr getConstBlob() const {
-        return constBlob;
-    }
+private:
+    void cloneBlobIfRequired(const MKLDNNDims& dims, const InferenceEngine::Precision& prec);
 
 private:
+    std::shared_ptr<ngraph::Node> origLayer;
```

Comment:
Seems like unused.

Reply:
It is necessary to hold a reference to the original buffer inside ngraph::Constant.